sunpy/cm/color_tables.py

```diff
@@ -329,8 +331,12 @@ def eit_color_table(wavelength):
         244.,  248.,  250.]), 255 * np.ones(181)))
 
 
-def lasco_color_table(number):
-    """Returns one of the fundamental color tables for SOHO LASCO images."""
+def sswidl_lasco_color_table(number):
+    """Returns one of the previously used color tables for SOHO LASCO images.
```

Comment:
Remove use of word "previous" as this will likely not mean much in a few years. Replace with SSW-defined tables.